src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/BDD.cs

```diff
@@ -423,6 +413,19 @@ long Get(int i)
 
             //the result is the last BDD in the nodes array
             return nodes[n - 1];
+
+            // Gets the i'th element from the underlying array of longs represented by bytes
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+            static long Get(int bytesPerLong, ReadOnlySpan<byte> bytes, int i)
```

Comment:
static is to make certain no captures creep in?

Reply:
yes

Reply:
(the span has to be passed in regardless as it can't be captured)